### PR TITLE
Add account

### PR DESCRIPTION
**Pull request by shreyasananth-3**

### CODE REVIEW

File: packages/cysync-core/src/dialogs/AddAccountGuide/Dialogs/index.ts [hunk 1]
```diff
@@ -1,3 +1,3 @@
 export * from './Asset';
-export * from './Device';
+export * from './X1 Vault';
```
Comment (irshadCypherock) [inline packages/cysync-core/src/dialogs/AddAccountGuide/Dialogs/index.ts]: Keep this `Device`, `X1 Vault` is only for display texts, not for folder and file names.

**Review by irshadCypherock — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/atoms/Checkbox.tsx [hunk 2]
```diff
@@ -61,6 +61,11 @@ const CheckBoxLabelStyle = styled.label.attrs<ISize>(props => ({
     left: ${({ size }) => (size === 'big' ? '2px' : '1.5px')};
     background-image: ${({ theme }) => theme.palette.background.sideBar};
   }
+
+  ${CheckBoxStyle}:focus + & {
+    outline: 2px solid ${({ theme }) => theme.palette.background.sideBar};
+    outline: 2px solid #eef209;
```
Comment (shreyasananth-3) [inline packages/ui/src/components/atoms/Checkbox.tsx]: As mentioned line 67 works but 66, even the style doesnt get applied. Need your help on whether am I making any mistake with respect to template literal @irshadCypherock

File: packages/ui/src/components/utils/index.ts [hunk 3]
```diff
@@ -55,4 +55,4 @@ export * from './height.styled';
 export * from './animate.styled';
 export * from './border.styled';
 export * from './Gradient';
-export * from './MenuFilter';
+export * from './dropdown.util';
```
Comment (irshadCypherock) [inline packages/ui/src/components/utils/index.ts]:
```suggestion
export * from './dropdown';
```

File: packages/ui/src/components/atoms/index.ts [hunk 4]
```diff
@@ -17,6 +17,6 @@ export * from './Select';
 export * from './LeanBoxContainer';
 export * from './Tag';
 export * from './Toggle';
-export * from './Dropdown';
+export * from './Dropdown/Dropdown';
```
Comment (irshadCypherock) [inline packages/ui/src/components/atoms/index.ts]:
```suggestion
export * from './Dropdown';
```

File: packages/ui/src/components/atoms/Dropdown/Dropdown.tsx [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+  items,
+  shouldShowIcon,
+  searchText,
+  placeholderText,
+  selectedItem = undefined,
+  onChange,
+  disabled = false,
+}) => {
+  const [search, setSearch] = useState('');
+  const [isOpen, setisOpen] = useState(false);
```
Comment (irshadCypherock) [inline packages/ui/src/components/atoms/Dropdown/Dropdown.tsx]:
```suggestion
  const [isOpen, setIsOpen] = useState(false);
```

File: packages/ui/src/components/atoms/Checkbox.tsx (hunk 2 above)
Comment (irshadCypherock) [reply packages/ui/src/components/atoms/Checkbox.tsx]: It won't work since `sideBar` is a gradient.

**Review by irshadCypherock — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/atoms/Checkbox.tsx (hunk 2 above)
Comment (shreyasananth-3) [reply packages/ui/src/components/atoms/Checkbox.tsx]: background.sidebar is hex value, its there in color.styled
    sidebar: '#1E1A15',

File: packages/ui/src/components/atoms/Dropdown/Dropdown.tsx (hunk 5 above)
Comment (shreyasananth-3) [reply packages/ui/src/components/atoms/Dropdown/Dropdown.tsx]: done

File: packages/ui/src/components/atoms/index.ts (hunk 4 above)
Comment (shreyasananth-3) [reply packages/ui/src/components/atoms/index.ts]: done

File: packages/ui/src/components/utils/index.ts (hunk 3 above)
Comment (shreyasananth-3) [reply packages/ui/src/components/utils/index.ts]: done

File: packages/cysync-core/src/dialogs/AddAccountGuide/Dialogs/index.ts (hunk 1 above)
Comment (shreyasananth-3) [reply packages/cysync-core/src/dialogs/AddAccountGuide/Dialogs/index.ts]: Done

File: packages/ui/src/components/atoms/Checkbox.tsx (hunk 2 above)
Comment (shreyasananth-3) [reply packages/ui/src/components/atoms/Checkbox.tsx]: Changed the logic and handled it differently

**Comment by shreyasananth-3** [comment]
Even after resolving conflict locally, its still not updating

**Comment by shreyasananth-3** [comment]
trying to see whether reopening its reflecting new changes

**Pull request closed without merging**